mongo/mongo.go

```diff
@@ -4,46 +4,63 @@ import (
 	"context"
 	"errors"
 	"github.com/ONSdigital/dp-areas-api/apierrors"
-	"github.com/ONSdigital/dp-areas-api/config"
 	"github.com/ONSdigital/dp-areas-api/models"
 	"github.com/ONSdigital/dp-healthcheck/healthcheck"
-	dpmongo "github.com/ONSdigital/dp-mongodb"
-	dpMongoHealth "github.com/ONSdigital/dp-mongodb/health"
+	dpMongoHealth "github.com/ONSdigital/dp-mongodb/v2/pkg/health"
+	dpMongoDriver "github.com/ONSdigital/dp-mongodb/v2/pkg/mongo-driver"
 	"github.com/ONSdigital/log.go/log"
 	"github.com/globalsign/mgo"
 	"github.com/globalsign/mgo/bson"
```

Comment:
Switch to `go.mongodb.org/mongo-driver/bson`

Reply:
updated it in the new commit.